README.md

```diff
@@ -49,8 +49,8 @@ For production, use a public MongoDB, and follow the same instructions above for
 Run the following commands:
 
 ```shell
-docker compose pull
-docker compose up --build
+docker-compose pull
+docker-compose up --build
```

Comment:
`docker compose` is the correct way to do this: 
![image](https://user-images.githubusercontent.com/43357585/231700982-c0a271c0-e5f1-4e20-92da-0cbf1afad444.png)

This image is taken from the official Docker documentation.

Reply:
Ok. Then the corresponding documentation for oteapi services should be updated.

Reply:
I referenced this comment in the OTEAPI Service issue [here](https://github.com/EMMC-ASBL/oteapi-services/issues/257#issuecomment-1545829315).